### PR TITLE
Move next-mdx from zeit/next-plugins to zeit/next.js

### PR DESCRIPTION
**Pull request by ijjk**
At request of @timneutkens I moved the `next-mdx` plugin from the next-plugins repo into Next.js. Also fixed small typo in README under setup.

### CODE REVIEW

File: packages/next-mdx/package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@zeit/next-mdx",
```
Comment (timneutkens) [inline packages/next-mdx/package.json]:
```suggestion
  "name": "@next/mdx",
```

:smirk:

File: packages/next-mdx/package.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  "version": "1.1.1-canary.3",
```
Comment (timneutkens) [inline packages/next-mdx/package.json]:
```suggestion
  "version": "0.0.1",
```

**Comment by timneutkens** [comment]
Let's add a License file too.

File: packages/next-mdx/index.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+module.exports = (pluginOptions = {}) => (nextConfig = {}) => {
+  const extension = pluginOptions.extension || /\.mdx$/
+
+  return Object.assign({}, nextConfig, {
+    webpack (config, options) {
+      if (!options.defaultLoaders) {
```
Comment (timneutkens) [inline packages/next-mdx/index.js]: I think this if statement can be removed by now 🤔

Comment (ijjk) [reply packages/next-mdx/index.js]: Yeah I think 3 major versions should be good to remove it 👍

**Review by timneutkens — COMMENTED** [review]
Let's create a new test suite in `test/integration`: `test/integration/plugin-mdx`, you can `cp -r test/integration/config test/integration/config`

File: packages/next-mdx/index.js (hunk 3 above)
Comment (timneutkens) [reply packages/next-mdx/index.js]: 🙌

**Review by timneutkens — APPROVED** [review]

**Pull request merged**